shared/rakefiles/entrypoint.rake

```diff
@@ -30,7 +30,7 @@ Rake::Task["clean"].enhance do
 end
 
 task :clobber_volumes => :set_vars do
-  ["secrets", "gcloud", "aws"].each do |app|
+  ["secrets", "gcloud"].each do |app|
```

Comment:
Can we avoid orphaning an `-aws` volume in all the places that currently have one?

Reply:
I added a tasks that checks if the volume exists and throw a warning. I think that destroying the volume automatically can be too much intrusive.

Reply:
I'm wondering - we destroy the volumes currently when the `clobber_volumes` gets called, why would it become too intrusive suddenly, especially when those are not used anymore?

Reply:
+1 for removing the aws volume instead of displaying warning. Or maybe allow user to do that, but fail the execution. IMO warnings are rarely effective if they don't break the workflow.